functions/src/services/user/saveUser.ts

```diff
@@ -0,0 +1,95 @@
+import * as admin from 'firebase-admin';
+import { UserSocial } from '../../models/userSocial.model';
+import { User } from '../../models/user.model';
+
+
+const saveUserSocialRecord = async (social: UserSocial) => {
+  try {
+    await admin.firestore().doc(`userSocial/${social.userId}`).set(social);
+  }
+  catch (err) {
+    console.log('Error in savingUserSocailRecord', err);
+  }
+}
+
+const saveUserRecord = async (record: User) => {
+  try {
+    await admin.firestore().doc(`user/${record.uid}`).set(record)
+  }
+  catch (err) {
+    console.log('Error in savingUserRecord', err);
+    throw err;
+  }
+}
+
+const getGithubStats = (user: User, social: UserSocial): UserSocial | null => {
```

Comment:
Not sure the `getGitHub...` functions make sense to be under `saveUser`

Reply:
Okay Eddie will move the functions to a separate file.